stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -195,13 +195,9 @@ public ClientIOTaskResult runLocal() throws Exception {
     Map<String, SummaryStatistics> summaryStatistics = new HashMap<>();
 
     Map<String, MethodStatistics> nameStatistics =
-        processMethodProfiles(startMs, endMs, (type, method) -> {
-          if ((type.equals("AlluxioBlockInStream") && method.equals("readChunk")) || (
-              type.equals("HDFSPacketReceiver") && method.equals("doRead")) || (
-              type.equals("HDFSBlockReaderRemote")
-                  && method.equals("newBlockReader")
-                  || method.equals("readChunk"))) {
-            return method;
+        processMethodProfiles(startMs, endMs, javaAgentInput -> {
+          if (javaAgentInput.getIsttfb()) {
```

Comment:
If the class was renamed, then this would probably be `profileInput`

Reply:
changed.